frontend/src/static/js/v_summary.js

```diff
@@ -137,8 +134,25 @@ window.vSummary = {
       const newSize = 100 * (slice.insertions / this.avgCommitSize);
       return Math.max(newSize * this.rampSize, 0.5);
     },
-    getSlicePos(i, total) {
-      return (total - i - 1) / total;
+    getCommitPos(i, total, sinceDate, untilDate) {
+      return (total - i - 1) / total * DAY_IN_MS
```

Comment:
the * and / its not clear whether `DAY_IN_MS` is in the numerator or denominator, add in the brackets

Reply:
should be `((total - i - 1) / total) * DAY_IN_MS`
but it is implicitly so
actually Eugene ever mentioned to drop brackets here haha